docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3496,19 +3594,44 @@ state parse_l2 {
 }
 ~ End P4Example
 
+The corresponding entry in the P4Info for this Value Set is:
+
+~ Begin Prototext
+value_sets {
+  preamble {
+    id: 0x03000001
+    name: "trill_types"
+  }
+  match {
+    id: 1
+    bitwidth: <ETH_TYPE_BITWIDTH>
+    match_type: EXACT
```

Comment:
Same comment as above on `match_type` being inside of another `match` message

Reply:
The other `match` message here is a oneof. It doesn't actually create a level of nesting, which is why the field numbering doesn't get a new scope. It is not exactly like a C union. What it means is that when serializing the message, at most one of the fields will be present.